jcl/src/java.base/share/classes/java/lang/J9VMInternals.java

```diff
@@ -387,6 +387,7 @@ private static void threadCleanup(Thread thread) {
 
 	/*[PR CVMC 124584] checkPackageAccess(), not defineClassImpl(), should use ProtectionDomain */
 	private static void checkPackageAccess(final Class clazz, ProtectionDomain pd) {
```

Comment:
While you're here, please fix the raw type use:
```java
	private static void checkPackageAccess(final Class<?> clazz, ProtectionDomain pd) {
```

Reply:
Updated